docs/reference/docs/data-replication.asciidoc

```diff
@@ -5,12 +5,12 @@
 [float]
 === Introduction
 
-Each index in Elasticsearch is <<getting-started-shards-and-replicas,divided into shards>>
+Each index in Elasticsearch is <<scalability,divided into shards>>
```

Comment:
```suggestion
Each index in {es} is <<scalability,divided into shards>>
```

Reply:
I was trying to leave the blinders on when I fixed the xref...

Reply:
Fair enough. Feel free to ignore. Not trying to inflate scope.

Reply:
Ah. Took another look. It looks like your text editor might be trimming whitespace, which makes the changes look larger than they are. My mistake.

I'll resolve any comments I have in data-replication.asciidoc.